debug/src/debug.js

```diff
@@ -15,6 +15,22 @@ import { assign, isNaN } from './util';
 
 const isWeakMapSupported = typeof WeakMap == 'function';
 
+/**
+ * @param {import('./internal').VNode} vnode
+ * @returns {Array<string>}
+ */
+function getDomChildren(vnode) {
+	let domChildren = [];
+	vnode._children.forEach(child => {
+		if (typeof child.type === 'function') {
+			domChildren = [...domChildren, ...getDomChildren(child)];
```

Comment:
This branch is probably not hit very often, so tuning this might not make a visible difference. That said we can skip the intermediate array creation with a neat little trick:

```suggestion
			domChildren.push.apply(domChildren, getDomChildren(child));
```

This is basically the ES5 equivalent to  `domChildren.push(...getDomChildren(child))`